pkg/sbom/sbom_interface.go

```diff
@@ -0,0 +1,7 @@
+package sbom
+
+type SbomClient interface {
+	GetSBOM(imageID string)
+	FilterSBOM(sbomFileRelevantMap map[string]bool)
+	PostFilterSBOM()
```

Comment:
[Question]: Post? Think of a different name.

Reply:
done